src/liballoc/arc.rs

```diff
@@ -331,7 +341,9 @@ impl<T> Arc<T> {
     }
 }
 
-impl<T: ?Sized> Arc<T> {
+impl<T: ?Sized, A: Alloc> Arc<T, A> {
+    // TODO: Do into_raw or from_raw need to know about Alloc?
```

Comment:
```
[00:04:07] tidy error: /checkout/src/liballoc/arc.rs:345: TODO is deprecated; use FIXME
```

Reply:
Yeah, I'll admit that I haven't tested this locally because of weird unrelated Mac issues (that I've seen references to elsewhere, but I'm hoping I can just get lucky and push a commit that works...). If this keeps failing, I'll figure out what's wrong with my setup.